templates/webhook.init.erb

```diff
@@ -7,6 +7,15 @@
 #
 # description: Enables web based runs of r10k i.e. github enterprise post hooks
 # processname: webhook
+#
+### BEGIN INIT INFO
+# Required-Start:    $remote_fs $network
+# Required-Stop:     $remote_fs $network
```

Comment:
I don’t think this is right. Were these parameters copied from another script?

Reply:
Do you mean the $remote_fs part?

To answer your question, yes I took inspiration from the other examples in /etc/init.d on my system.  Almost every script that depended on $network also depended on $remote_fs, and I thought it not impossible that webhook would also be used with a network filesystem.

Reply:
Oh, I did not realize those were variables for systemd. Carry on.

Reply:
LSB tags only apply to init scripts, not systemd. (Took some time to find that out).

Unless Debian/Ubuntu are doing something with systemd that no one else is, this is only applicable to the sysVInit scripts.

That being said, this appears to be the template for the init script, not the systemd unit file, so I have no issues with this change. Especially since it's primarily a documentation/non-functional and non-breaking change